recipes/cppcolormap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cppcolormap" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "1b6e0213454e6a7432281224841466363c330d0c343ac1cbea4640c1be77fa27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tdegeus/cppcolormap/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - python
+    - setuptools
+    - xtensor
+    - pyxtensor
```

Comment:
Most of these are not build deps.

Reply:
@scopatz Thanks for your review. The recipe provides two things at once: a header only C++ library and and Python module with access to all the C++ functions from Python. These dependencies are build dependencies for the Python module only. The C++ library has no build dependencies (other than CMake and a compiler to avoid complaints from CMake).

I was wondering, and your question is in this direction, if it would be better to split it into two recipes (`cppcolormap` and `python-cppcolormap` or `cppcolormap-python`) to stay more clean on the dependency side. It would be good to clarify this as I have a few more libraries like this that I want to share.

Reply:
Right they are not build deps, they are host deps

Reply:
because you link to them on install

Reply:
Thanks ! I somehow misunderstood both your comment and the host section 😞

Reply:
Yeah, sorry, I don't have a ton of time do do these reviews so I am sometimes too terse!

Reply:
No worries. I'm quite impressed by the quality and speed of the reviews you guys get around doing. Also it seems you're not that many... Anyway, thanks again!